kratos/utilities/helper_classes_for_constraint_builder.h

```diff
@@ -143,7 +181,7 @@ class AuxiliaryGlobalMasterSlaveConstraint : public IndexedObject
     void SetLeftHandSide(const double LhsValue)
     {
         mLockObject.SetLock();
-            mLhsValue = LhsValue;
+        mLhsValue = LhsValue;
```

Comment:
A tab here was intentional to distinguish which part of code in inside the locked region. Any way .. this will go soon .. !

Reply:
Sorry